### PR TITLE
[MM-32278] Change "E20 Only" tag on listings to "E20 and Cloud"

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Should it be "E20 and Cloud" or "Cloud and E20"?

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-32278

### CODE REVIEW

File: internal/model/label.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ var BetaLabel Label = Label{
 }
 
 var EnterpriseLabel Label = Label{
-	Name:        "Enterprise (E20)",
-	Description: "This plugin only works on servers with an E20 license.",
+	Name:        "Enterprise (E20 and Cloud)",
```
Comment (aaronrothschild) [inline internal/model/label.go:30]:
```suggestion
	Name:        "Enterprise (Cloud or E20)",
```

File: internal/model/label.go [hunk 2]
```diff
@@ -27,7 +27,7 @@ var BetaLabel Label = Label{
...
+	Description: "This plugin only works on servers with an E20 license or on Mattermost Cloud.",
```
Comment (aaronrothschild) [inline internal/model/label.go:31]:
```suggestion
	Description: "This plugin only works on servers with an E20 license or with a Mattermost Cloud Instance.",
```
@justinegeffen do you know if "Instance" is how we are referring to a customer's "Cloud Server"?

Comment (justinegeffen) [reply internal/model/label.go:31]: That's a very good question. Let me look into that.

File: internal/model/label.go [hunk 3]
```diff
@@ -27,7 +27,7 @@ var BetaLabel Label = Label{
 }
 
 var EnterpriseLabel Label = Label{
-	Name:        "Enterprise (E20)",
-	Description: "This plugin only works on servers with an E20 license.",
+	Name:        "Enterprise (Cloud or E20)",
```
Comment (justinegeffen) [inline internal/model/label.go:30]:
```suggestion
	Name:        "Enterprise Edition E20" (Cloud or self-managed),
```

File: internal/model/label.go [hunk 4]
```diff
@@ -27,7 +27,7 @@ var BetaLabel Label = Label{
...
+	Description: "This plugin only works on servers with an E20 license or on Mattermost Cloud.",
```
Comment (justinegeffen) [inline internal/model/label.go:31]:
```suggestion
	Description: "This plugin only works on self-hosted deployments using an Enterprise Edition E20 license and Mattermost Cloud workspaces.",
```

File: internal/model/label.go (hunk 2 above)
Comment (justinegeffen) [reply internal/model/label.go:31]: They're called workspaces; I've added my suggestion. :) 

Note on this: Does this work on Cloud Free, Professional, and Enterprise?

File: internal/model/label.go (hunk 3 above)
Comment (justinegeffen) [reply internal/model/label.go:30]: I don't know whether the suggested change affects any UI elements.

**Review by aaronrothschild — APPROVED** [review]

File: internal/model/label.go (hunk 3 above)
Comment (hanzei) [reply internal/model/label.go]: Screenshot from 2021-02-04 14-14-23

@justinegeffen That is a bit to long :wink: . Can you think of something short?

Comment (justinegeffen) [reply internal/model/label.go]: I'm 0/5 on it and happy for it to remain the same. :)

File: internal/model/label.go [hunk 5]
```diff
@@ -27,7 +27,7 @@ var BetaLabel Label = Label{
 }
 
 var EnterpriseLabel Label = Label{
-	Name:        "Enterprise (E20)",
-	Description: "This plugin only works on servers with an E20 license.",
+	Name:        "Enterprise Edition E20 (Self-managed and Mattermost Cloud)",
+	Description: "This plugin only works on self-managed deployments using an Enterprise Edition E20 license and Mattermost Cloud workspaces.",
```
Comment (justinegeffen) [inline internal/model/label.go:31]:
```suggestion
	Description: "This plugin only works on self-managed deployments (E20) and Mattermost Cloud workspaces.",
```

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thanks, @hanzei, LGTM! :)

**Comment by DHaussermann** [comment]
@hanzei while testing https://mattermost.atlassian.net/browse/MM-32278 I noticed that this change does indded work as expected. However your change was in master and is not yet in the production branch. 

For now I have adjusted fix version. I'm unclear on the process for Merging an individual feature such as this yo the production branch. If there a follow-up PR comming?

**Comment by hanzei** [comment]
Yes, I will merge `master` into `production` after https://github.com/mattermost/mattermost-marketplace/pull/170 is merged.

**Pull request merged**